contrib/win32/win32compat/ssh-agent/connection.c

```diff
@@ -31,6 +31,11 @@
 #include "agent.h"
 #include "agent-request.h"
 
+#ifdef ENABLE_PKCS11
+#include "ssh-pkcs11.h"
+#define ENABLE_PKCS11
```

Comment:
Can you define ENABLE_PKCS11 in https://github.com/PowerShell/openssh-portable/blob/latestw_all/contrib/win32/openssh/config.h.vs

Reply:
Line 36 is hit only when ENABLE_PKCS11 is defined? Is it necessary?

Reply:
I see this is already defined in https://github.com/PowerShell/openssh-portable/blob/991465e07974ac194e35a77c032f9c7337f37cf5/contrib/win32/openssh/config.h.vs#L138